libs/designsystem/src/lib/components/calendar/calendar.component.ts

```diff
@@ -146,21 +185,38 @@ export class CalendarComponent implements OnInit, AfterViewInit, OnChanges {
     return !!this.yearNavigatorOptions;
   }
 
-  constructor(private calendarHelper: CalendarHelper, @Inject(LOCALE_ID) private locale: string) {
-    moment.locale(this.locale);
-    moment().format('YYYY-MM-DD');
+  constructor(private calendarHelper: CalendarHelper, @Inject(LOCALE_ID) locale: string) {
+    this.locale = this.mapLocale(locale);
+    this.timeZoneName = Intl.DateTimeFormat().resolvedOptions().timeZone;
+  }
+
+  // 'PP' format is Apr 29, 1453
+  private formatWithLocale(date: Date, formatString = 'PP'): string {
```

Comment:
As far as I can see all method calls supply a formatString? Why the default? 🤔 
```suggestion
  private formatWithLocale(date: Date, formatString: string): string {
```

Reply:
Excellent question 😁  
I think it made sense a few refactorings back - I'll remove the default